src/lib/components/multiDropzone/index.test.tsx

```diff
@@ -60,6 +60,21 @@ describe('MultiDropzone component', () => {
       expect(screen.getByText("Too many files.")).toBeVisible();
     });
 
+    it("should show max file size error message", async () => {
+      const screen = setup({maxSize: 10 });
+      const input = screen.getByTestId(inputTestId);
+      const bigFile = file;
+      Object.defineProperty(bigFile, 'size', { value: 1024 });
+
+      await act(async () => {
+        fireEvent.change(input, { target: { files: [bigFile] } });
```

Comment:
_This comment does not require any changes in this PR_
I think we'll have to update `testing-library` and add `user-event` pretty soon. As noted in the testing-library documentation, [using fireEvent is not recommended and instead we should use user-event](https://testing-library.com/docs/user-event/intro#differences-from-fireevent).
Can you create a follow up ticket for tackling this in the future?

Reply:
Noted and 100% agree!